themes/bootstrap3/templates/myresearch/profile.phtml

```diff
@@ -84,6 +84,42 @@
     <?php endif; ?>
   </div>
 
+  <?php $user = $this->auth()->isLoggedIn(); ?>
+  <?php if ($this->auth()->getManager()->supportsPersistentLogin($user)): ?>
+    <div class="myresearch-profile-header">
+      <h3><?=$this->transEsc('Saved logins')?></h3>
+    </div>
+    <?php $tokens = $user->getLoginTokens($user->id); ?>
+    <?php if (count($tokens) === 0): ?>
+      <div class="my-profile-col"><?=$this->transEsc('No saved logins')?></div>
+    <?php else: ?>
+      <table class="login-tokens-table table table-striped">
+        <thead>
+          <tr>
+            <th scope="col" class="login_token_browser"> <?=$this->transEsc('Platform')?> / <?=$this->transEsc('Browser')?></th>
+            <th scope="col" class="login_token_last_login"><?=$this->transEsc('Last Login')?>:</th>
+            <th scope="col" class="login_token_delete"><?=$this->transEsc('Delete')?></th>
```

Comment:
Having a "Delete" heading on a column containing only "Forget Login" buttons seems a little redundant. Would a more general heading like "Actions" make more sense?

Might there also be a benefit to a "Forget All" button to quickly clear everything out?

For accessibility purposes, should we put labels on the buttons to help differentiate them from one another (e.g. by including more details in addition to the "Forget Login" text)?

Reply:
Sure, "Actions" does sound better. I'll also add a "Forget All" button.

For accessibility, I'll add the platform/browser and login time info to the label